modules/core/src/main/java/org/apache/ignite/events/CacheConsistencyViolationEvent.java

```diff
@@ -67,39 +68,58 @@
     private static final long serialVersionUID = 0L;
 
     /** Represents original values of entries.*/
-    final Map<Object, Map<ClusterNode, EntryInfo>> entries;
+    final Map<?, Map<ClusterNode, EntryInfo>> entries;
```

Comment:
Looks like "?" is not required in this class at all.
Have no clue why it was required before, but this was made because of compilation issues.
Thanks for pointing this.